### PR TITLE
Add guide for {{each-in}}

### PR DESCRIPTION
**Pull request by tomdale**

### CODE REVIEW

File: source/templates/displaying-the-keys-in-an-object.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      <li>WhistlePig</li>
+      <li>High West</li>
+    </ol>
+  </li>
+</ul>
+```
+
+### A Note About Automatic Updates
+
+Note that the `{{#each-in}}` helper does **not** observe property
```
Comment (trek) [inline source/templates/displaying-the-keys-in-an-object.md]: trolling me?

Comment (rwjblue) [inline source/templates/displaying-the-keys-in-an-object.md]: Dang, sorry didn't see your comment when I merged. I can revert if you'd like?

Comment (rwjblue) [inline source/templates/displaying-the-keys-in-an-object.md]: I think the concept that a rerender is needed when you add props is important information IMHO. Can we remove the 'Note that' lead in from this sentence?

Comment (tomdale) [inline source/templates/displaying-the-keys-in-an-object.md]: ABT — Always Be Trolling

**Pull request merged**